Sources/SwiftToolchain/Builder.swift

```diff
@@ -49,12 +49,12 @@ public final class Builder {
     self.fileSystem = fileSystem
   }
 
-  public func run() -> AnyPublisher<String, Error> {
+  private func processPublisher(builderArguments: [String]) -> AnyPublisher<String, Error> {
     let buildStarted = Date()
     let process = ProcessRunner(
-      arguments,
+      builderArguments,
       loadingMessage: "Compiling...",
-      parser: DiagnosticsParser(),
+      parser: nil,
```

Comment:
FYI @carson-katri I'm disabling this parser for now since it sometimes fails to parse project errors, but lets through the GTK warning. This causes confusing diagnostic messages as reported in https://github.com/swiftwasm/carton/issues/241. A better long-term solution would be to implement https://github.com/swiftwasm/carton/issues/181 instead, I hope to get to it at some point, unless someone else is willing to take this over.